components/gitpod-db/src/typeorm/entity/db-workspace.ts

```diff
@@ -121,4 +121,10 @@ export class DBWorkspace implements Workspace {
     })
     @Index("ind_basedOnSnapshotId")
     basedOnSnapshotId?: string;
+
+    @Column({
+        default: "",
+        transformer: Transformer.MAP_EMPTY_STR_TO_UNDEFINED,
+    })
+    workspaceClass?: string;
```

Comment:
We already have that field on the WorkspaceInstance. Why do we need to add it here, now, as well? :thinking: